_data/domains.yml

```diff
@@ -320,9 +320,10 @@ websites:
     - name: Moniker
       url: https://www.moniker.com/
       img: moniker.png
-      tfa: No
-      twitter: monikersnap
-      facebook: MonikerLLC
+      tfa: Yes
+      software: Yes
+      email: Yes
```

Comment:
Looks like they only support 2FA through software 2FA. Email seems to only be authorization.
```suggestion
```